code/game/machinery/vending/vendor_types/crew/staff_officer.dm

```diff
@@ -1,5 +1,154 @@
 /obj/structure/machinery/cm_vending/clothing/staff_officer
-	name = "\improper Broken ColMarTech Staff Officer Equipment Rack"
-	desc = "An automated equipment vendor for Staff Officers. Sadly, it doesn't work."
-	req_access = list(777)//ACCESS_MARINE_BRIDGE
+	name = "\improper ColMarTech Staff Officer Equipment Rack"
+	desc = "An automated equipment vendor for Staff Officers."
+	req_access = list(ACCESS_MARINE_COMMANDER)
 	vendor_role = list(JOB_SO)
+
+/obj/structure/machinery/cm_vending/clothing/staff_officer/get_listed_products(var/mob/user)
+	return GLOB.cm_vending_clothing_staff_officer
+
+//------------GEAR---------------
+
+GLOBAL_LIST_INIT(cm_vending_clothing_staff_officer, list(
+		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
+		list("Boots", 0, /obj/item/clothing/shoes/marine/knife, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),
+		list("M3-VL Light Vest", 0, /obj/item/clothing/suit/storage/marine/light/vest, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
```

Comment:
this is combat equipment (armour that it makes no sense to be mandatory)

Reply:
are you saying to remove it or remove it as mandatory